### PR TITLE
[WIP] Add jenkins jobs for e2e test

### PR DESCRIPTION
**Pull request by hangyan**
Signed-off-by: Hang Yan <yhang@vmware.com>

### CODE REVIEW

File: ci/jenkins/test-flow-visibility.sh [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    echo "=== grafana-clickhouse-datasource is NOT installed correctly ==="
+    FAILURE=true
+  fi
+}
+
+config_antrea() {
+  echo "=== Stopping Antrea === "
+  kubectl delete -f ${GIT_CHECKOUT_DIR}/build/yamls/antrea.yml
+  echo "=== Enable Antrea Flow Exporter ==="
+  sed -i -e "s/#  FlowExporter: false/  FlowExporter: true/g" ${GIT_CHECKOUT_DIR}/build/yamls/antrea.yml
```
Comment (wsquan171) [inline ci/jenkins/test-flow-visibility.sh:100]: we also need to change the following params in antrea.yaml to make the records to be exported sooner to be compatible with #14 
```
flowCollector:
  flowPollInterval: "1s"
  activeFlowExportTimeout: "2s"
  idleFlowExportTimeout: "1s"
```

Comment (hangyan) [reply ci/jenkins/test-flow-visibility.sh:100]: Ok

Comment (hangyan) [reply ci/jenkins/test-flow-visibility.sh:100]: @wsquan171 I have updated the settings and runs the e2e test code. The clickhouse result seems still have some delays, which caused the related test failed. Any thoughts? 

Here is some output from the test 


test log:
```bash
=== RUN   TestFlowAggregator/IPv4/IntraNodeFlows
    flowaggregator_test.go:913: Network Policies are realized.

    flowaggregator_test.go:870: 
        	Error Trace:	flowaggregator_test.go:870
        	            				flowaggregator_test.go:612
        	            				flowaggregator_test.go:605
        	            				flowaggregator_test.go:209
        	Error:      	Received unexpected error:
        	            	timed out waiting for the condition
        	Test:       	TestFlowAggregator/IPv4/IntraNodeFlows
        	Messages:   	ClickHouse did not receive the expected records in query output: ; query: SELECT * FROM flows WHERE (sourceIP = '192.168.0.2') AND (destinationIP = '192.168.0.3') AND (sourceTransportPort = 58454)
=== RUN   TestFlowAggregator/IPv4/IntraNodeDenyConnIngressANP
I0507 13:44:51.684314 3040944 framework.go:583] Creating/updating Antrea NetworkPolicy antrea-test/test-flow-aggregator-anp-ingress-reject
I0507 13:44:51.703286 3040944 framework.go:583] Creating/updating Antrea NetworkPolicy antrea-test/test-flow-aggregator-anp-ingress-drop
```

actual result from clickhouse db:

```bash
2022-05-07 13:46:16     2022-05-07 13:45:10     2022-05-07 13:45:23     2022-05-07 13:45:23     2022-05-07 13:45:23     3       192.168.0.2     192.168.0.3     58454   5201    6       356     15091837        356     15091837        249     12956   249       12956   perftest-a      antrea-test     jenkins-theia-e2e-for-pull-request-61-dtnrg     perftest-b      antrea-test     jenkins-theia-e2e-for-pull-request-61-dtnrg     0.0.0.0 0               test-flow-aggregator-networkpolicy-ingress-allow antrea-test              1       1       test-flow-aggregator-networkpolicy-egress-allow antrea-test             1       1       TIME_WAIT       1                       6824272 5824    6824272 6824272 5824    5824    0
2022-05-07 13:46:24     2022-05-07 13:45:10     2022-05-07 13:45:23     2022-05-07 13:45:23     2022-05-07 13:45:23     3       192.168.0.2     192.168.0.3     58454   5201    6       356     15091837        0       0       249     12956   0       0perftest-a       antrea-test     jenkins-theia-e2e-for-pull-request-61-dtnrg     perftest-b      antrea-test     jenkins-theia-e2e-for-pull-request-61-dtnrg     0.0.0.0 0               test-flow-aggregator-networkpolicy-ingress-allow        antrea-test               1       1       test-flow-aggregator-networkpolicy-egress-allow antrea-test             1       1       TIME_WAIT       1                       0       0       0       0       0       0       0
```

**Comment by hangyan** [comment]
close due to visiablity change.

**Pull request closed without merging**